test/Google.Api.Gax.Grpc.Tests/CallSettingsExtensionsTest.cs

```diff
@@ -337,5 +337,53 @@ public void WithExpiration_SettingsWithRetry()
             Assert.Same(backoffSettings, result.Timing.Retry.TimeoutBackoff);
             Assert.Equal(token, result.CancellationToken);
         }
+
+        [Fact]
+        public void WithResponseMetadataHandler_OnlyHandler()
+        {
+            Metadata setByHandler = null;
+            var settings = ((CallSettings) null).WithResponseMetadataHandler(m => setByHandler = m);
+            var metadata = new Metadata();
+            settings.ResponseMetadataHandler(metadata);
+            Assert.Same(setByHandler, metadata);
```

Comment:
I think the args should be swapped. `metadata` is the expected value? ([Assert.Same()](https://github.com/xunit/assert.xunit/blob/bbece92b18661cc18ede9d72b28648466f9a1709/IdentityAsserts.cs#L32))